docs/content/examples/tutorials/basic-installation.md

```diff
@@ -25,37 +25,37 @@ We are going to use this docker based mailserver:
     !!! example
 
         ```yaml
-        version: '2'
+        version: '3.8'
 
         services:
-        mail:
-            image: mailserver/docker-mailserver:latest
+          mailserver:
+            image: docker.io/mailserver/docker-mailserver:latest
             hostname: mail
-            domainname: example.org
-            container_name: mail
+            domainname: example.com
+            container_name: mailserver
             ports:
-            - "25:25"
-            - "587:587"
-            - "465:465"
+                - "25:25"
+                - "587:587"
+                - "465:465"
             volumes:
-            - ./data/:/var/mail/
-            - ./state/:/var/mail-state/
-            - ./config/:/tmp/docker-mailserver/
-            - /var/ds/wsproxy/letsencrypt/:/etc/letsencrypt/
+                - ./data/:/var/mail/
+                - ./state/:/var/mail-state/
+                - ./config/:/tmp/docker-mailserver/
+                - /var/ds/wsproxy/letsencrypt/:/etc/letsencrypt/
             environment:
-            - PERMIT_DOCKER=network
-            - SSL_TYPE=letsencrypt
-            - ONE_DIR=1
-            - DMS_DEBUG=1
-            - SPOOF_PROTECTION=0
-            - REPORT_RECIPIENT=1
-            - ENABLE_SPAMASSASSIN=0
-            - ENABLE_CLAMAV=0
-            - ENABLE_FAIL2BAN=1
-            - ENABLE_POSTGREY=0
+                - PERMIT_DOCKER=network
+                - SSL_TYPE=letsencrypt
+                - ONE_DIR=1
+                - DMS_DEBUG=0
+                - SPOOF_PROTECTION=0
+                - REPORT_RECIPIENT=1
```

Comment:
By the way, this is deprecated, what should it be changed to ?
The docs are not clear about that

Reply:
What exactly is deprecated?

Reply:
https://docker-mailserver.github.io/docker-mailserver/v10.0/config/environment/#report_recipient-deprecated

`REPORT_RECIPIENT` :)

Reply:
https://docker-mailserver.github.io/docker-mailserver/v10.0/config/environment/#pflogsumm_recipient

`PFLOGSUMM_RECIPIENT` or leave it blank

Reply:
The space count seems wrong (4 instead of 2) for the ports, volumes and environment items.

Reply:
I have already addressed indent issues in my commits. This file in particular gets it's own separate PR due to it's history and issues with it in general I've resolved, that will follow after my bulk changes PR.